jsondb/src/test/java/io/syndesis/jsondb/impl/JsonDBTest.java

```diff
@@ -209,9 +209,9 @@ public void testPush() throws IOException {
     @Test
     public void testCreateKey() {
         String lastkey = jsondb.createKey();
-        for (int i = 0; i < 20000; i++) {
+        for (int i = 0; i < 200000; i++) {
```

Comment:
I know, its not related to this PR but since you touched this test, I believe this test can be removed completely as the same test is used in KeyGeneratorTest and `JsonDB.createKey()` just delegates to this method.

Reply:
Done.